library/reals/lisp-reals.lisp

```diff
@@ -0,0 +1,30 @@
+;;;; lisp-reals.lisp
+;;;;
+;;;; Lisp backend for Reals library using computable reals (https://github.com/stylewarning/computable-reals)
+
+(defpackage #:coalton-library/lisp-reals
+  (:use #:cl)
+  (:local-nicknames (#:cr #:computable-reals))
+  (:export
+   #:*creal-comparison-threshold*
+   #:into-creal))
+
+(in-package #:coalton-library/lisp-reals)
+
+(defvar *creal-comparison-threshold* 106)
+
+;;;
+;;; Converting objects into creals
+;;;
+
+(defgeneric into-creal (x)
```

Comment:
Don't define generic functions to call from Coalton. Just call the implementation directly in the instance definition.